prisma/schema.prisma

```diff
@@ -241,23 +242,25 @@ model NewsComment {
 }
 
 model Recruitment {
-  id                 String    @id @default(cuid())
-  archived           Boolean   @default(false)
-  createdAt          DateTime  @default(now()) @map("created_at")
-  updatedAt          DateTime  @default(now()) @updatedAt @map("updated_at")
-  companyName        String    @map("company_name")
-  website            String
-  job                String
-  address            String
-  position           String
-  title              String
-  description        String
-  type               String    @map("type")
-  startAt            DateTime? @map("start_at")
-  expiredAt          DateTime? @map("expired_at")
-  salary             String    @default("negotiate")
-  isApproved         Boolean   @default(false)
-  recruitmentOwnerId String    @map("recruitment_owner_id")
+  id                     String       @id @default(cuid())
+  archived               Boolean      @default(false)
+  createdAt              DateTime     @default(now()) @map("created_at")
+  updatedAt              DateTime     @default(now()) @updatedAt @map("updated_at")
+  companyName            String       @map("company_name")
+  website                String
+  job                    String
+  address                String
+  position               String
+  title                  String
```

Comment:
what is the difference between `job` and `title`?

Reply:
![image](https://user-images.githubusercontent.com/73436010/220316338-98d8b8c8-8384-47c9-b7e3-7045f45116f4.png)
as you can see, job is your major career, and your title is the name of the recruitment